src/table/cells/selections.jsx

```diff
@@ -9,23 +8,30 @@ const useStyles = makeStyles({
     background: 'repeating-linear-gradient(-45deg, #fafafa, #fafafa 2px, #eee 2.5px, #fafafa 3px, #fafafa 5px)',
   },
   selected: {
-    color: '#fff',
+    color: '#fff !important',
     backgroundColor: '#009845',
   },
   possible: {},
 });
 
-export function withSelections(CellComponent) {
+export default function withSelections(CellComponent) {
```

Comment:
The reason there was no `default` here we to be able to mock it for testing. If you can make it work with default, we can keep it, otherwise it shouldn't be there

Reply:
now it works with default